mlflow/fastai.py

```diff
@@ -380,7 +380,9 @@ def load_model(model_uri):
 
 @experimental
 @autologging_integration(FLAVOR_NAME)
-def autolog(log_models=True, disable=False, exclusive=False):  # pylint: disable=unused-argument
+def autolog(
+    log_models=True, disable=False, exclusive=False, disable_for_untested_versions=False
```

Comment:
When a user calls `mlflow.autolog()`, `mlflow.fastai.autolog()`, etc with `disable_for_untested_versions=False`, can we still emit a warning message if the user is using an untested / unsupported library version via `_logger.warn`? It would be great if this message also encouraged users to either upgrade MLflow or downgrade the ML library to a documented supported version if they encounter errors during autologging.

The rationale behind this warning message is that users may be unaware of this flag because it defaults to `False`, so a warning message will help users interpret errors / failures.

Reply:
Anther question is when disable_for_untested_versions=True, and meet unsupported version, should we also print message say it is unsupported, or just silently log nothing ?